tests/check/check_h26xsigned_sign.c

```diff
@@ -580,6 +577,7 @@ START_TEST(recurrence)
           ck_assert_int_gt(item->data_size, last_sei_size);
         }
       }
+      last_sei_size = item->data_size;
```

Comment:
You have still removed the entire change from master. The test on master is more robust. Please add it back.